gdal-sys/build.rs

```diff
@@ -244,11 +242,25 @@ fn main() {
                 "cargo:rustc-cfg=gdal_sys_{}_{}_{}",
                 version.major, version.minor, version.patch
             );
-
+            let target_arch = std::env::var("CARGO_CFG_TARGET_ARCH").expect("Set by cargo");
+            let is_windows = std::env::var("CARGO_CFG_WINDOWS").is_ok();
+            let ptr_size = std::env::var("CARGO_CFG_TARGET_POINTER_WIDTH").expect("Set by cargo");
+
+            let binding_name = match (target_arch.as_str(), ptr_size.as_str(), is_windows) {
+                ("x86_64" | "aarch64", "64", false) => "gdal_x86_64-unknown-linux-gnu.rs",
+                ("x86_64", "64", true) => "gdal_x86_64-pc-windows-gnu.rs",
+                ("x86" | "arm", "32", false) => "gdal_i686-unknown-linux-gnu.rs",
+                ("x86", "32", true) => "gdal_i686-pc-windows-gnu.rs",
+                _ => panic!(
+                    "No pre-built bindings available for target: {} ptr_size: {} is_windows: {}",
+                    target_arch, ptr_size, is_windows
+                ),
+            };
             let binding_path = PathBuf::from(format!(
-                "prebuilt-bindings/gdal_{}_{}.rs",
-                version.major, version.minor
+                "prebuilt-bindings/{}.{}/{binding_name}",
```

Comment:
Nit: I can't test it right now, but I don't know if Explorer shows directory extensions by default. Maybe we should stick to `{}_{}`.

Reply:
I've just moved the files in https://github.com/lnicola/gdal/pull/1/commits/f49cb3a7918af9af69e5022720835e4b2f508933